auth_tkt/compat.py

```diff
@@ -0,0 +1,23 @@
+from base64 import b64decode, b64encode
+
+import six
+
+
+def base64encode(s, encoding='utf-8'):
+    if six.PY2:
+        return b64encode(s)
+    return b64encode(bytes(s, encoding)).decode()
```

Comment:
You might be able to make this `b64encode(s.encode(encoding)).decode(encoding)` on both python 2 and 3?

But then it'll need to always be using unicode strings, not byte strings on python 2.